lib/Objects/basic_device.dart

```diff
@@ -0,0 +1,54 @@
+/// Copyright 2020 Google LLC
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     https://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+class BasicDevice {
```

Comment:
Should explain what Basic Devic is. For ex: the object holds the device response object which includes all metadata excepts the model, meid, orderNumber, willAutoRenew, osVersion, platformVersion, firmwareVersion, macAddress, and bootMode properties. See the document from https://developers.google.com/admin-sdk/directory/v1/reference/chromeosdevices/get

Reply:
Done, resolving conversation.